### DISCUSSION TITLE
Not able to add new proxy host after 0.12.0 update.

### DISCUSSION THREAD

**Discussion by dnomode** [hassio-addons/addon-nginx-proxy-manager#342] [Q&A] [answered]
Hello, First of all, this is all kind of new to me, so if I'm missing something obvious, I apologize. After updating to version 0.12.0 many of the areas of Nginx Proxy Manager are no longer accessible. The only thing I can navigate to is the dashboard and Proxy Hosts. 

Access List, SSL Certtificates, Users, Audit Log, and Settings do nothing when clicked. I first noticed the issue when trying to add a New Proxy Host. I'm not 100% sure if the update had something to do with this or something else. The current Proxy Hosts were setup when I was on 0.11.0 , I updated to 0.12.0 and then yesterday I tried to add a new proxy host and noticed things not working as they should. 

What should I do to fix? Please help. Thanks

Ed

**Comment by dnomode** [comment, marked as answer]
I figure this out. Its something to do with Firefox. Today i decided to try with Safari and everything works as it should.

Thanks

**Comment by johanf85** [comment]
Had the same issue today, didn't work in Chrome or Firefox. Did work in Safari. Was using is via external access on domain:81
